docs/tut-making-jettycat.md

```diff
@@ -287,22 +294,22 @@ this.tex = ct.random.dice(
 );
 ```
 
-`ct.random.dice` is a function that accepts any number of arguments and returns one of them randomly at each time it is called.
+`ct.random.dice` is a function that accepts any number of arguments and returns one of them randomly each time it is called.
 
 Time for testing! If your pipes spawn misaligned, check that you set up collision shapes for all the four textures and put their axis to the bottom of a pipe.
 
 ## Spawning pipes through time
 
 Like templates, rooms can have their own logic as well — they are hidden under the button "Room events" in a room editor. There are four events as well:
```

Comment:
There are also timers as well, and action events. Maybe this paragraph and list should be removed or rephrased.